ARTIFACTORY_API_SUPPORT.md

```diff
@@ -0,0 +1,64 @@
+# Artifactory API support
+
+Artipie supports artifactory API to manage repositories, users and repository permissions.
+
+## Create Repository
+
+Creates new docker repository with `default` storage alias and provided name.
+
+> **PUT** /api/repositories/{repoKey}
+
+Consumes json with the following fields (any other fields are ignored): 
+
+Field name | Type | Meaning | Required
+------------ | ------------- | ------------ | ---------
+key | string | New repository name | Y
+rclass | string | Repository class, only `local` type is supported | Y
+packageType | string | Artifact type, only `docker` type is supported | Y
+dockerApiVersion | string | Docker API version, we only support `V2` | Y
+
+Possible responses:
+- `200 OK` when new repository was successfully created
+- `400 BAD REQUIEST` in the cases when repository with such name already exists or invalid json was sent
+- `500 INTERNAL ERROR` in the case of unexpected server error
+
+## Users
+
+### Get the users list
```

Comment:
@olenagerasimova title should be capitalized to be in line with other titles in the file and general formatting rules for English texts